skopt/plots.py

```diff
@@ -280,6 +297,13 @@ def plot_objective(result, levels=10, n_points=40, n_samples=250, size=2,
         Scale to use for the z axis of the contour plots. Either 'linear'
         or 'log'.
 
+    * `objective_name` [str, default='Partial dependence']
+        Label of the objective function to show on y-axis.
+
+    * `dim_names` [list of str, default=None]
+        Labels of the dimension variables. `None` defaults to `['X_0', 'X_1',
```

Comment:
How about "Select a subset of dimensions from the search space and plot only those. By default all dimensions are plotted."?